src/components/SidebarNav/SidebarNav.tsx

```diff
@@ -0,0 +1,116 @@
+import '../../index.css'
+import React from 'react'
+import { Dropdown } from '@components/Dropdown/Dropdown'
+import TrickerLogo from '@assets/TrickerLogo'
+import TrickerTitle from '@assets/TrickerTitle'
+import { NavbarItem } from '@components/NavbarItem/NavbarItem'
+import Icon from '@components/Icon/Icon'
+import TimeTrackingBadge from '@components/TimeTrackingBadge/TimeTrackingBadge'
+import { ProfileButton } from '@components/ProfileButton/ProfileButton'
+import Body1 from '@utils/typography/body1/body1'
+import { NavLink } from 'react-router-dom'
+import { type TimeTracking, type User } from '@utils/types'
+
+export interface SidebarNavProps
+  extends React.HTMLAttributes<HTMLInputElement> {
+  user: User
+  variant: 'pm' | 'dev'
+  timeTracking?: TimeTracking
+  dropdownOptions: Array<{ title: string; image: string }>
+  handleDropdownSelect: (option: { title: string; image: string }) => void
+}
```

Comment:
The design is correct. Regarding the 'variant' prop to handle the state when the user is a project manager or not in the project, I think it would be better for the data-flow to be as follows:

- Fetch information from the backend about all the projects the user is working on. As the endpoint to do this does not exist yet, we could leave the structure set up or prepared to add it when it becomes available.
- Store that information in a global context to avoid having to repeatedly query the backend.
- Pass those projects to the Dropdown component when it's called. It would receive an array of UserProjectRole[] as props (accessing the field UserProjectRole.project.name will give us the project name, and accessing UserProjectRole.project.image will give us the project image).
- The selected project (via the Dropdown) should bring us information about the user's role so that the sidebar can determine which view to display.

Taking these details into account, the data flow becomes much clearer, and the responsibility of the states or the data to be received would be tied to the component rather than the user, as it should be.